### ISSUE TITLE
Type archived web pages as such

### ISSUE THREAD

**Issue by acka47** [hbz/lobid#152]
See https://github.com/hbz/lobid/issues/99#issuecomment-66102018 and following comments.

From https://github.com/hbz/lobid/issues/99#issuecomment-106256832:

> We may get the information whether something is an archived web page from MAB field 051, element 1, see e.g. http://lobid.org/resource?id=TT002234459&format=source where there's a `w` in element 1:
`<controlfield tag="051">mw||||||</controlfield>`.

**Comment by acka47** [comment]
@jschnasse Have you heard something re. `w` in field 051 as indicator for an archived web page?

**Comment by acka47** [comment]
Ping @jschnasse. Let us know if you know more about the `w` in 051.

**Comment by dr0i** [comment]
@aquast acknowledged in https://github.com/hbz/lobid/issues/242#issuecomment-235254511 that `051w` indicates the archived web page.

**Comment by dr0i** [comment]
@aquast in #313:

> Speaking with LBZ staff it was clarified that indicator w in position 1 at field 051 can be used to identify web sites. **Therefore interpeting field 655 can be skipped in association with edoweb maybe?**

This seems to be not ok, look e.g. the [publisso ellinet-resource HT018801101](http://lobid.org/resource?id=HT018801101&format=source) where `051w` is missing but `652a1` has the value `Archivierte Online-Ressource`.  
Also there are (horrible?) catalog entries like e.g. [edoweb HT018801101](http://lobid.org/resource?id=HT018801101&format=source) where `051.y` is used (whatever THAT means - it's not mentioned in the [table]https://github.com/hbz/lobid/issues/242#issuecomment-154030161) and three redundant times `655e1=http://edoweb-rlp.de/resource/edoweb:1750751` is used (one for the www-domain, one without www, and one for https. Well, at least the distinction with or without `www` should not be needed, and also the `https` addendum should be superflous).

In short: `051.w` doesn`t seem to be always there when it's about an `archivedWebPage`. (This whole `655` thing is totally broken, misused by a generation of librarians who don't know nothing about controlled vocabularies and their own storage systems. Shame. Cataloging is broken :( )

**Comment by jschnasse** [comment]
"Archivierte Online-Ressource" is not a type statement! In fact nearly any bibliographic resource can be archived online. The statement "Archivierte Online-Ressource" just states that a digital and online manifestation of the book, report, thesis, etc. is available.

**Comment by dr0i** [comment]
Douh , you are right, we [already have had this discussion 1.5 years ago](https://github.com/hbz/lobid/issues/99#issuecomment-88018532).

**Comment by dr0i** [comment]
Deployed to production. See http://lobid.org/resources/TT002234459 and http://lobid.org/resource/TT002234459. Close if pleased.

**Comment by acka47** [comment]
I wonder whether we should remove the `bibo:Book` typing from archived web pages. What do you think, @jschnasse ?

**Comment by jschnasse** [comment]
Indeed a 'w' in MAB50 should win over 'm'. 'm' = "Eigenständiges Werk". 'w' = "Webseite". I assume 'w' is always used in this combination!? The mapping of 'mw' to "bibo:book" sounds wrong to me.

**Comment by aquast** [comment]
Thank you for the implementation. From my perspective bibo:book should be removed as it's a wrong typ for websites. I wonder also, if typ shuold be website rather than archivedWebpage as we have no archivedBook or something else. As far as I understood ZB MED and LBZ colleagues I agree with Jan

**Comment by acka47** [comment]
@dr0i, please remove `bibo:Book` typing from archived web pages.

**Comment by dr0i** [comment]
Removed. Will be deployed to staging at sunday, production at monday. Please review then.

**Comment by acka47** [comment]
I am pleased and I guess also will be @aquast and @jschnasse. Closing.
